fastify.js

```diff
@@ -70,7 +83,31 @@ function build (options) {
       return
     }
 
-    router(stripUrl(req.url), req, res)
+    runInSeries(
+      new State(req, res),
+      _onRequestIterator,
+      onRequestRaw(),
+      routeCallback
+    )
+  }
+
+  function State (req, res) {
+    this.req = req
+    this.res = res
+  }
+
+  function _onRequestIterator (fn, cb) {
+    setImmediate(fn, this.req, this.res, cb)
```

Comment:
Because we have seen that when performing a lot of async operations, call the callbacks with `setImmediate` helps to improve the performances.
Do you think this is not the case?